src/main/java/com/synopsys/integration/detect/configuration/enumeration/BlackduckScanMode.java

```diff
@@ -3,6 +3,7 @@
 public enum BlackduckScanMode {
     RAPID ("Rapid"),
     EPHEMERAL ("Ephemeral"),
```

Comment:
The release note says the EPHEMERAL value is deprecated; calling  EnumProperty.newBuilder().deprecateValue() here will actually deprecated it.

Reply:
If I knew that...